markdown/developers/adding_modules.md

```diff
@@ -255,7 +255,7 @@ Please follow the steps below to run the tests locally:
 
 3. Install [`pytest-workflow`](https://pytest-workflow.readthedocs.io/en/stable/#installation)
 
-4. Start running your own tests using the appropriate [`tag`](https://github.com/nf-core/modules/blob/3d720a24fd3c766ba56edf3d4e108a1c45d353b2/tests/modules/fastqc/test.yml#L3-L5) defined in the `test.yml`:
+4. Start running your own tests using the appropriate [`tag`](https://github.com/nf-core/modules/blob/master/tests/modules/fastqc/test.yml#L3-L4) defined in the `test.yml`:
```

Comment:
How come this needed switching, the blob URL should be pretty stable, no? I try to avoid using `master` as those lines could say something completely different when clicked on in the future..

Reply:
I was also wondering that...maybe because the commit, actually didn't contain `test.yml`, but was just the last commit back then? I will change it to a blob url with the last commit with changes to test.yml:

```suggestion
4. Start running your own tests using the appropriate [`tag`](https://github.com/nf-core/modules/blob/20d8250d9f39ddb05dfb437603aaf99b5c0b2b41/tests/modules/fastqc/test.yml) defined in the `test.yml`:
```